azure-pipelines.yml

```diff
@@ -177,6 +177,15 @@ jobs:
           pip install $_.FullName
       }
     displayName: 'Build NumPy'
+  - bash: |
+      pushd . && cd .. && target=$(python -c "import numpy, os; print(os.path.abspath(os.path.join(os.path.dirname(numpy.__file__), '.libs')))") && popd
```

Comment:
we don't have Python 2 in development Windows Azure CI so careful with this on backport I suppose

Reply:
Actually, I conditionally restricted this to Python 3.6 recently, so the usage of `print()` should only be "ok".